build_tools/python/benchmark_suites/iree/armv8_a_benchmarks.py

```diff
@@ -24,6 +24,10 @@ class Android_ARMv8_A_Benchmarks(object):
         tf_models.GPT2_117M_1x1_FP32_TF,
         tflite_models.MOBILEBERT_INT8,
         tflite_models.VIT_INT8_TFL,
+        matmul.MATMUL_1x256x2048_I8xI4_MLIR,
+        matmul.MATMUL_256x256x2048_I8xI4_MLIR,
+        matmul.MATMUL_1x256x2048_I8xI8_MLIR,
+        matmul.MATMUL_256x256x2048_I8xI8_MLIR,
```

Comment:
These models will be tested with local-sync, 1 and 2 threads. Maybe we want to add them separately and only test with local-sync?

Reply:
Changed to use local-sync